src/kernel.jl

```diff
@@ -36,6 +36,14 @@ function product2d(kf)
     k1[1].*k1[2], k2[1].*k2[2]
 end
 
+"""
+    kern = box(m, n)
+    kern = box((m, n, ...))
+
+Return a box kernel computing a moving average. `m, n, ...` specify the size of the kernel, which is centered around zero.
+"""
+box(sz::Dims) = broadcast(*, KernelFactors.box(sz)...)
```

Comment:
```suggestion
box(sz::Dims) = broadcast(*, KernelFactors.box(sz)...)
box(sz::Int...) = box(sz)
```

Reply:
Sadly we can't really do that. Others are specialized to assume 2d with a single input :frowning_face: .

Reply:
Hmmm, then I guess we need to remove the `kern = box(m, n)` signature from the docstring?

Reply:
Oh, right, thanks!